src/quo2/components/list_items/saved_address/component_spec.cljs

```diff
@@ -0,0 +1,79 @@
+(ns quo2.components.list-items.saved-address.component-spec
+  (:require [test-helpers.component :as h]
+            [quo2.components.list-items.saved-address.view :as saved-address]
+            [quo2.foundations.colors :as colors]))
```

Comment:
I see tests for many styles in this file. I think we were talking about that sometime earlier that we shouldn't be testing styles? 🤔 cc @ilmotta @J-Son89

Reply:
Yeah, actually, I just wanted to test backgroundColor, not the entire style, but I didn't find a better way to do it. Of course I am open to change the approach if you have any suggestion, or to remove these tests entirely if they don't add too much value. Honestly, I don't have a strong opinion about this and I am not 100% sure about these tests.

Reply:
Yeah, I remember those discussions @OmarBasem. I tend to almost never say, "don't test that" because usually if the developer went into the trouble of writing a unit test that was useful for them to solve a problem, that probably means it's going to be valuable to other devs in the future.

But having said that, if I remember correctly, the idea was to test via Jest only, no CLJS tests directly for the style namespace. I'm totally okay with that decision, although I still think in rare occasions it's fine to write (CLJS) unit tests for some more complicated style functions.

Reply:
Updated to just check for `backgroundColor` which is the only style prop that needs to be tested 👍

Reply:
sure, all sounds good to me 👍